src/applications/personalization/profile-2/components/ProfileWrapper.jsx

```diff
@@ -102,6 +106,12 @@ const mapStateToProps = state => {
   // or fails:
   const hasLoadedMilitaryInformation = state.vaProfile?.militaryInformation;
 
+  // when the call to load MHV fails, `errors` will be set to a non-null value
+  // when the call succeeds, the `accountState` will be set to a non-null value
+  const hasLoadedMHVInformation =
+    selectProfile(state)?.mhvAccount?.errors ||
+    selectProfile(state)?.mhvAccount?.accountState;
```

Comment:
I _think_ we need to check for `mhv` in the user profile's `services`, like checking for `evss`, if that isn't happening already

Reply:
oo, thank you. I'll check.

Reply:
I'm actually not seeing a check for that in the existing AccountApp/AccountMain components.

Reply:
In fact I don't see this value used anywhere in the app https://github.com/department-of-veterans-affairs/vets-website/blob/952cfa79232e26c3b3153708a0fd0c781be3a280/src/platform/user/profile/constants/backendServices.js#L21

Reply:
Ah looks like it's divided up into a few services right now. There probably aren't any references to the values right now because MHV hasn't been used by any apps since vets.gov. Are there any services that seem to fit your use case?

Reply:
Well I'm not seeing a check for any specific services being available before making the call to `fetchMHVAccount()` in the existing Account section (which uses `AccountApp` and `AccountMain`)

Reply:
Nvm, seeing as how AccountApp doesn't do it, it probably isn't necessary